README.md

```diff
@@ -1,2 +1,11 @@
 # docker-spark-gpg
-Docker Alpine image with Spark, Python, Java and gpg. Used for local dev environment.
+Docker Alpine image with Spark, Python, Java and gpg. Used for loacal dev environment.
+
+Instructions
+
+7. Run the command in order to mount the export data folder on to the root of the Docker container
+`docker run -v ~/export-data:/root/export-data -v ~/export-data/gpg:/root/gpg -it docker-spark-gpg :latest bash`
```

Comment:
I know it's only an example, but got should not be a sub dir of export-data 
Shouldn't there be a mount point for the output?